### PR TITLE
New project-less image build

### PR DESCRIPTION
**Pull request by sneridagh**
I'd like to start testing it with things :) 

I called them "-next" suffix, naming... I've thought about "-project-less" but it's also ugly.

I can imagine that both should be around for a while...

### CODE REVIEW

**Comment by stevepiercy** [comment]
Do you need to rubber duck a name for this thingie? "project-less image" is not an informative name.

**Comment by davisagli** [comment]
One of the defining characteristics of the new approach is that it uses pnpm instead of yarn, right? So maybe `-pnpm` for the name?

I have mixed feelings about publishing both. If this is meant to be considered experimental for some period of time, that's not very obvious.

File: next/pnpm-workspace.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+packages:
+  - 'core/packages/*'
+  - 'packages/volto-light-theme'
```
Comment (ericof) [inline next/pnpm-workspace.yaml:3]: Should this be here?

**Comment by davisagli** [comment]
This is out of date. Replaced by https://github.com/kitconcept/plone-frontend/pull/1 and https://github.com/plone/plone-frontend/pull/47

**Pull request closed without merging**